torch/quantization/quantize_script.py

```diff
@@ -35,7 +35,6 @@ def _prepare_script(model, qconfig_dict, inplace=False, quant_type=QuantType.STA
     if not all(isinstance(x, str) for x in qconfig_dict.keys()):
         raise ValueError('qconfig_dict should only contain names(str) as keys.')
     scripted_qconfig_dict = script_qconfig_dict(qconfig_dict)
-    torch._C._jit_pass_dedup_module_uses(model._c)
```

Comment:
Dont you need to remove the corresponding tests?

Reply:
this is a legacy pass we added for this case, but this case can be addressed with current code already. I feel this pass might not be needed if we have some (unlikely) case like:

```python
x = self.submodule_with_states(x)
...
x = self.submodule_with_states(x)
```

that is: we use some submodule with states multiple times.

I only removed this from API, this pass is still there. If we decide to support the above use case at some point, we'll debug the problem and re-enable the pass.